src/useq/_mda_event.py

```diff
@@ -98,6 +104,8 @@ class MDAEvent(UseqModel):
     z_pos : float | None
         Z position in microns. If not provided, implies use current position. By
         default, `None`.
+    autofocus : tuple[str, float] | None
```

Comment:
"When not None, consumers of this event must perform a hardware autofocus as follows: ..."